Realm/RealmSwift/Object.swift

```diff
@@ -70,3 +52,24 @@ public class Object : RLMObjectBase, Equatable {
 public func == <T: Object>(lhs: T, rhs: T) -> Bool {
     return lhs.isEqualToObject(rhs)
 }
+
+public class ObjectUtil : NSObject {
```

Comment:
If I make it private it seems to be inaccessible from objective-c. Perhaps there is another way?